go.mod

```diff
@@ -0,0 +1,53 @@
+module github.com/kitex-contrib/config-file
+
+go 1.19
+
+require (
+	github.com/cloudwego/kitex v0.7.2
+	github.com/cloudwego/kitex-examples v0.2.0
```

Comment:
Please make the subdirectory `example` an individual module by adding a `go.mod`, thus we don't need to require `kitex-examples` here.